frontend/assets/style/components/_combobox.scss

```diff
@@ -1,38 +1,26 @@
-.combobox {
+.selectsolo {
```

Comment:
Note: A combo field in the web usually means it is a select box where each child has multiple elements, (ex: a list of countries with flags) or it has some built it functionality (ex: a search within the select list).

This element is just a simple select box that can be used solo in the page (ex: nr of items per page). That's why I think `.selectsolo` is a better name than `.combobox`.